opensafely/jupyter.py

```diff
@@ -13,6 +13,18 @@
 DESCRIPTION = "Run a jupyter lab notebook using the OpenSAFELY environment"
 
 
+# poor mans debugging because debugging threads on windows is hard
+if os.environ.get("DEBUG", False):
+    def debug(msg):
+        # threaded output for some reason needs the carriage return or else
+        # it doesn't reset the cursor.
+        sys.stderr.write("DEBUG: " + msg.replace("\n", "\r\n") + "\r\n")
+        sys.stderr.flush()
+else:
+    def debug(msg):
+        pass
```

Comment:
Given the function name and parameters are the same for both forks, is it worth pushing the if-block inside the function?

Something like:

```
def debug(msg):
    DEBUG = os.environ.get("DEBUG", False)
    if not DEBUG:
        return

    # true branch
    …
```

Reply:
Why do 20 `if` s if you can do one!

Reply:
Also, I plan to DTRT thing an add debug logging and -vvv switches at somepoint, so this is only temporary